package.json

```diff
@@ -31,6 +31,7 @@
     "ember-ajax": "2.0.1",
     "ember-autosave": "2.0.1",
     "ember-bootstrap": "0.7.2",
+    "ember-browserify": "^1.1.13",
```

Comment:
Preprints does not seem to be building without ember-browserify now if pointed towards the Ember-OSF PR for the latest discover page changes https://github.com/CenterForOpenScience/ember-osf/pull/205